src/main/java/com/exadel/sober/controllers/AddictionController.java

```diff
@@ -0,0 +1,25 @@
+package com.exadel.sober.controllers;
+
+import com.exadel.sober.repositories.AddictionRepository;
+import com.exadel.sober.models.Addiction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin
+@RequestMapping(path="/addiction")
+public class AddictionController {
+    @Autowired
+    private AddictionRepository addictionRepository;
+
+    @PostMapping(path="/add")
```

Comment:
Don't use things like "add" "remove" and so on... If you are doing POST to the AddicitionController - that means you are trying to create an Addiction, id DELETE - you are truing to delete and so on. So type of the request and name of controller must be enough to understand why you are trying to do

Reply:
done